CHANGELOG

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+Unreleased
+==========
+
+- Misc:
+
+  - doc: extend description on how to use the configuration file
```

Comment:
Thanks for taking care of the changelog, but I do not think small doc improvements are worth an entry there.